components/edit-collective/EditPayPalAccount.js

```diff
@@ -0,0 +1,155 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useMutation } from '@apollo/react-hooks';
+import { useFormik } from 'formik';
+import { FormattedMessage } from 'react-intl';
+
+import { GraphQLContext } from '../../lib/graphql/context';
+import { API_V2_CONTEXT, gqlV2 } from '../../lib/graphql/helpers';
+
+import { getI18nLink } from '../I18nFormatters';
+import StyledButton from '../StyledButton';
+import StyledInput from '../StyledInput';
+import StyledInputField from '../StyledInputField';
+import { P } from '../Text';
+
+const createConnectedAccountMutation = gqlV2`
+  mutation createConnectedAccount($connectedAccount: ConnectedAccountCreateInput!, $account: AccountReferenceInput!) {
+    createConnectedAccount(connectedAccount: $connectedAccount, account: $account) {
+      id
+      settings
+      service
+      createdAt
+      updatedAt
+    }
+  }
+`;
+
+const deleteConnectedAccountMutation = gqlV2`
+  mutation deleteConnectedAccount($connectedAccount: ConnectedAccountReferenceInput!) {
+    deleteConnectedAccount(connectedAccount: $connectedAccount) {
+      id
+    }
+  }
+`;
+
+const mutationOptions = { context: API_V2_CONTEXT };
+
+const EditPayPalAccount = props => {
+  const { refetch } = React.useContext(GraphQLContext);
```

Comment:
I'm not familiar with that, which queries does it refetch? What's the benefit of using that over `refetchQueries` or an `update` function?

Reply:
LOL! Thanks for that!
![image](https://user-images.githubusercontent.com/2119706/82827933-67a69c80-9e86-11ea-99e5-f57a90d20312.png)